### ISSUE TITLE
Junos: error in trunk interface parsing in case no vlan is given

### ISSUE THREAD

**Issue by jean1** [pdav/netmagis#159]
When parsing the following config (actual config used in ethernet ring protection):

```
interfaces {
    xe-0/0/1 {
        description X;
        unit 0 {
            family ethernet-switching {
                port-mode trunk;
            }
        }
    }

```
we get the following error message:
```
can't read "k(*)": no such element in array
    while executing
"set l $k(*)"
    (procedure "juniper-parse-list" line 25)
    invoked from within
"juniper-parse-list kwtab [lindex $conf 1] t "$idx!vlans!name!$nom""
    (procedure "juniper-parse-vlans-entry" line 15)
    invoked from within
"$fct $conf t $idx"
    (procedure "juniper-parse-list" line 54)
    invoked from within
"juniper-parse-list kwtab [lindex $conf 1] t $idx"
    (procedure "juniper-parse-vlans" line 11)
    invoked from within
"$fct $conf t $idx"
    (procedure "juniper-parse-list" line 54)
    invoked from within
"juniper-parse-list kwtab $conf t "eq!$eq""
    (procedure "juniper-parse" line 31)
    invoked from within
"$type-parse $libdir $modele $fdin $fdout t $eq"
    (procedure "parse" line 6)
    invoked from within
"parse $libdir $type $modele $fd stdout tab $eq"
    (procedure "main" line 26)
    invoked from within
"main $argv0 $argv"
    invoked from within
"exit [main $argv0 $argv]"
    (file "./analyze" line 576)

```

**Comment by jean1** [comment]
Problem is not under the interface level but at the top level in the vlans definition. The relevant configuration that produce the parsing error is:

```
vlans {
    my-cluster {
        vlan-id 1021;
        interface {
            xe-0/0/3.0;
            xe-0/0/4.0;
            xe-0/0/2.0;
        }
    }
```

**Comment by jean1** [comment]
closed by commit 92e6982b
